styles/globals.css

```diff
@@ -2,4 +2,9 @@
 
 @tailwind components;
 
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+@font-face {
```

Comment:
Wouldn't it be better to use the @layer directive of Tailwind here (https://tailwindcss.com/docs/adding-base-styles)? I haven't integrated a new font using Tailwind before, but I guess that would be the way to go.